### DISCUSSION TITLE
How to write nodal stress to anim files ?

### DISCUSSION THREAD

**Discussion by drozp94** [OpenRadioss/OpenRadioss#572] [Q&A] [answered]
Hello,

I have difficulties to write the nodal stress (or nodal strain for example) to the animation files,
those animation files i am converting to vtk and doing the visualization in ParaView.

The commands below will export the elemental stress or elemental strain values but not nodal.

/ANIM/SHELL/TENS/STRESS/ALL
/ANIM/SHELL/TENS/STRAIN/ALL
 
I've tried to find the solution in documentation, but didn't had a success from a first trial,
maybe somebody of you know the answer.

Thank you!
Dima

**Comment by sebastienVilleneuve** [comment, marked as answer]
Hi Dima,

You should try to use /ANIM/GPS/STRESS/TENS & /ANIM/GPS/STRAIN/TENS options.

Thanks,
Sébastien
